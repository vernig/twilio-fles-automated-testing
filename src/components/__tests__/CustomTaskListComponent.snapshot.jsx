import React from 'react';
import renderer from 'react-test-renderer';

import CustomTaskList from '../CustomTaskList/CustomTaskList';

describe('CustomTaskListComponent Snapshot', () => {
  it('compare demo component to snapshot', (done) => {
    const props = {
      isOpen: true,
      dismissBar: () => undefined,
    };
    const tree = renderer.create(<CustomTaskList {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
    done();
  });
});
