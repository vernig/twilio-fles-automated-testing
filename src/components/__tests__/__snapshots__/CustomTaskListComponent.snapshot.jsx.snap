// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomTaskListComponent Snapshot compare demo component to snapshot 1`] = `
<div
  className="css-157ejlk"
>
  This is a dismissible demo component
  <i
    className="accented"
    onClick={[Function]}
  >
    close
  </i>
</div>
`;
